dev/release/verify-release-candidate.sh

```diff
@@ -382,6 +382,11 @@ install_go() {
     return 0
   fi
 
+  if which go > /dev/null 2>&1; then
+    show_info "Found $(go version) at $(which go)"
```

Comment:
Should we also set ` GO_ALREADY_INSTALLED=1` ?

Reply:
That is being set at the end of the function when we install go manually. 

We could set nonetheless, just following the pattern from the other functions.